apps/emqx/src/emqx_map_lib.erl

```diff
@@ -31,7 +31,7 @@
         ]).
 
 -export_type([config_key/0, config_key_path/0]).
--type config_key() :: atom() | binary() | string().
+-type config_key() :: atom() | binary() | [byte()].
```

Comment:
New checks found an actual type error, config key can't be an utf8 list, since list_to_binary function is used which doesn't handle unicode.